wallet/src/Pos/Wallet/Web/Tracking/Sync.hs

```diff
@@ -503,19 +503,19 @@ getEncInfo encSK = do
     let wCId = encodeCType $ makeRootPubKeyAddress pubKey
     (hdPass, wCId)
 
-selectOwnAccounts
+selectOwnAddresses
     :: (HDPassphrase, CId Wal)
     -> (a -> Address)
     -> [a]
     -> [(a, CWAddressMeta)]
-selectOwnAccounts encInfo getAddr =
-    mapMaybe (\a -> (a,) <$> decryptAccount encInfo (getAddr a))
+selectOwnAddresses encInfo getAddr =
+    mapMaybe (\a -> (a,) <$> decryptAddress encInfo (getAddr a))
 
 setLogger :: HasLoggerName m => m a -> m a
 setLogger = modifyLoggerName (<> "wallet" <> "sync")
 
-decryptAccount :: (HDPassphrase, CId Wal) -> Address -> Maybe CWAddressMeta
-decryptAccount (hdPass, wCId) addr = do
+decryptAddress :: (HDPassphrase, CId Wal) -> Address -> Maybe CWAddressMeta
+decryptAddress (hdPass, wCId) addr = do
```

Comment:
I don't get why these changes are in this commit.

Reply:
hz, I noticed and decided to fix it